runtime/lua/vim/treesitter/dev.lua

```diff
@@ -400,6 +410,13 @@ function M.inspect_tree(opts)
       treeview:draw(b)
     end,
   })
+  api.nvim_buf_set_keymap(b, 'n', 'R', '', {
+    desc = 'Toggle between vim range and Range',
```

Comment:
I don't think most users know the difference between these. I think "Toggle between 0 and 1-based indexing" would be clearer.